<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI_load_Excel</class>
 <widget class="QWidget" name="GUI_load_Excel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_fileaddr">
       <property name="text">
        <string>Excel file address</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_fileaddr"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_fileaddr">
       <property name="text">
        <string>Select from system</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2,1,2">
     <item>
      <widget class="QLabel" name="label_template">
       <property name="text">
        <string>Template worksheet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_template">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_template">
       <property name="text">
        <string>User manualy</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_template"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2,1,2">
     <item>
      <widget class="QLabel" name="label_worksheet">
       <property name="text">
        <string>Active worksheet     </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_worksheet">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_worksheet">
       <property name="text">
        <string>User manualy</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_worksheet"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_script">
       <property name="text">
        <string>Script files directory</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_script"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_script">
       <property name="text">
        <string>Select from system</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_config">
       <property name="text">
        <string>Config files directory</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_config"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_config">
       <property name="text">
        <string>Select from system</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_flagQR">
     <property name="text">
      <string>Write results in table directly after test complete</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_flagCLR">
     <property name="text">
      <string>Automatic clear user sequances (before changes and after test complete)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_flagBR">
     <property name="text">
      <string>Write results after user break</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_noload_excwndw">
     <property name="text">
      <string>Do not load excel window after start</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_breakaftermiss">
     <property name="text">
      <string>Break test if returned value is out of range</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="Load">
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
